tgui-next/packages/tgui/interfaces/NtosSupermatterMonitor.js

```diff
@@ -48,9 +48,9 @@ export const NtosSupermatterMonitor = props => {
                 minValue={0}
                 maxValue={5000}
                 ranges={{
-                  good: [-Infinity, 5000],
-                  average: [5000, 7000],
-                  bad: [7000, Infinity],
+                  good: [-Infinity, 1],
+                  average: [1, 1.4],
```

Comment:
Well, this is exactly why I don't like this change, because 7000 has a meaning here, 1.4 is a meaningless ratio.